specification/subscription/resource-manager/readme.md

```diff
@@ -71,6 +71,18 @@ description: The subscription client
 
 ### Tag: package-2019-10-preview
 
+These settings apply only when `--tag=package-2023-05-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2023-05-preview'
+input-file:
+- Microsoft.Subscription/stable/2021-10-01/subscriptions.json
+- Microsoft.Subscription/preview/2023-05-01-preview/subscriptions.json
+title: SubscriptionClient
+description: The subscription client
```

Comment:
This seems wrong, as tag 2023-05-01-preview is being under comment "### Tag: package-2019-10-preview". Please confirm.